Calculation.java

```diff
@@ -1,3 +1,15 @@
+import java.util.Scanner;
+
+public class Calculation {
+    public static void main (String args[]){
+        Scanner scan = new Scanner(System.in);
```

Comment:
create a method "subtraction" in the existing class & remove the new class & new main method